libraries/AP_HAL_ChibiOS/hwdef/HolybroG4_Airspeed/hwdef-bl.dat

```diff
@@ -0,0 +1,84 @@
+# hw definition file for processing by chibios_pins.py
+
+# MCU class and specific type
+MCU STM32G4xx STM32G474xx
+
+FLASH_RESERVE_START_KB 0
+FLASH_BOOTLOADER_LOAD_KB 32
+
+# reserve some space for params
+APP_START_OFFSET_KB 4
+
+# 512k flash part
+FLASH_SIZE_KB 512
+
+# board ID for firmware load
+APJ_BOARD_ID 5405
```

Comment:
Needs a patch to board_types.txt

Reply:
"AP_HW_Holybro-ASP-G4                 5405"  has been added to file  board_types.txt.
Thank you very much!